### ISSUE TITLE
Styles not working in version 1.3.1

### ISSUE THREAD

**Issue by ezefire** [docstrap/docstrap#315]
When creating the doc I see css files are created under a directory with the same name and before the last update all css files were under styles directory directly.

**Comment by xeroxzone** [comment]
Also can confirm this in usage at https://github.com/xeroxzone/equivalent-js

`Uncaught SyntaxError: Unexpected token <
(index):1 Uncaught SyntaxError: Unexpected token <
(index):1 Uncaught SyntaxError: Unexpected token <
(index):214 Uncaught ReferenceError: $ is not defined
    at (index):214
(anonymous) @ (index):214
(index):285 Uncaught ReferenceError: $ is not defined
    at (index):285`

seems like the jquery resource is broken.

**Comment by lukeapage** [comment]
Hrmm sorry maybe there was a mistake when I published 1.3.1. I’m afraid I can’t look till tomorrow, but will try to resolve soon

**Comment by dolow** [comment]
I think #316 fixes this issue.

**Comment by lukeapage** [comment]
published 1.3.2, hope that works for you

**Comment by sfratini** [comment]
I was having the same issue and I can confirm 1.3.2 fixes it for me. Just one note, I had to delete my older doc folder and generate it again, because it couldn't run correctly due to this:

```

Generating output files...
fs.js:648
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: EISDIR: illegal operation on a directory, open 'C:\XXXXXXXX\docs\gen\fonts\glyphicons-halflings-regular.eot'

```

**Comment by xeroxzone** [comment]
Yes also can confirm its working again. Thanks!

**Comment by chharvey** [comment]
Yes! I posted the same issue over at https://github.com/mlucool/gulp-jsdoc3/issues/48. Looks like they need to update to v1.3.2.
